static/src/stylesheets/module/signup/_newsletters.scss

```diff
@@ -59,6 +59,8 @@
     background-color: $guardian-brand;
     height: 0;
     padding-bottom: 60%;
+    // Temporarily hiding until there are numerous images, to prevent repetition of icon
+    display: none;
```

Comment:
I would rather this not be in this PR - as all of the other changes I would approve.

As I understand there isn't likely to be different photos anytime soon (if ever) so we might as well not serve the html.